start.sh

```diff
@@ -65,4 +65,4 @@ fi
 echo "===================================="
 echo "Starting geniusyield-server..."
 set -x
-cabal run geniusyield-server -- serve
+/DEX/geniusyield-server -- serve
```

Comment:
This appears incorrect to me, I would expect `/DEX/geniusyield-server serve` instead.